src/components/RadioButtons/RadioButtons.jsx

```diff
@@ -13,17 +13,19 @@ import dispatchAnalyticsEvent from '../../helpers/analytics';
  * A radio button group with a label.
  *
  * Validation has the following props.
-
- * `additionalFieldsetClass` - String for any additional fieldset classes.
- * `additionalLegendClass` - String for any additional legend classes.
- * `errorMessage' - String Error message for the radio button group
- * `label` - String for the group field label.
- * `name` - String for the name attribute.
- * `tabIndex` - Number for keyboard tab order.
- * `options` - Array of options to populate group.
- * `required` - is this field required.
- * `value` - string. Value of the select field.
- * `onValueChange` - a function with this prototype: (newValue)
+ *
+ * - `additionalFieldsetClass` String for any additional fieldset classes.
+ * - `additionalLegendClass` String for any additional legend classes.
+ * - `errorMessage` String Error message for the radio button group
+ * - `label` String for the group field label.
+ * - `name` String for the name attribute.
+ * - `tabIndex` Number for keyboard tab order.
+ * - `options` Array of options to populate group.
+ * - `required` is this field required.
+ * - `value` string. Value of the select field.
+ * - `onValueChange` a function with this prototype: (newValue)
+ * - `ariaDescribedby` Array matching the options, added when the option is
+ *   selected
```

Comment:
Shows props in a list instead of all on one line

Reply:
Not related to this PR, but the `options` control is showing `[object Object]`
<img width="726" alt="Screen Shot 2021-06-03 at 11 32 28 AM" src="https://user-images.githubusercontent.com/136959/120680332-ccb0ad00-c45f-11eb-94d1-9ac5967b3036.png">

Reply:
Hmm, this makes me wonder if I should overload the `options` property:

```js
options={[
  { value: 'yes', label: 'Yes', ariaDescribedby: 'yes-id' },
  { value: 'no', label: 'No', ariaDescribedby; 'no-id' },
]}
```